### DISCUSSION TITLE
how to create multiline chart in react js

### DISCUSSION THREAD

**Discussion by mzulfanw** [d3/d3#3783] [Q&A] [answered]
hai everyone i'm stuck while want to create multiline chart with d3 js, i've been seek into observablehq but i'm still confuse, please help me

**Comment by Fil** [comment, marked as answer]
From the D3 gallery[1] you can open the multi-line chart example[2]

[1] https://observablehq.com/@d3/gallery
[2] https://observablehq.com/@d3/multi-line-chart/2?intent=fork

**Reply by mzulfanw** [reply]
@Fil let me try it, thanks for your answer

**Reply by mzulfanw** [reply]
in my current project is using d3.v5 its is work also for v5 ?

**Reply by curran** [reply]
Why not upgrade to the latest D3? All the patterns probably work with v5 as well. Did you try running the example with v5? What errors did you see?

**Reply by mzulfanw** [reply]
i'm currently working on legacy project which is using d3 v5
